### ISSUE TITLE
Image/Text Recognition

### ISSUE THREAD

**Issue by franciscoandreso** [Thomas-Fritz1/TeamProject#8]
Hello everyone,

Some of us are working with the image and text recognition using Unity. There are many tutorials on the internet about this, where it usually gives you an introduction for unity. There are also some problems that you can find in the way and must of them are caused because of the software version. 

I will share the tutorial I found, maybe you guys can have a look on that too.
https://www.youtube.com/watch?v=qfxqfdtxyVA
Here you will learn how recognize an image and to create a 3D object above that. I consider we can take the idea and apply it to the project. Not necessarily we need to put a 3D object on the thing we recognize but we can display more information about it.

Neeraja and Suraj had also made some progress on this. She told me about another tutorial with this link:
https://www.youtube.com/watch?v=qahu9PYaEbM

I face some troubles with the camera beacuse I saw only a black screen when I execute the application. There are some methods to do it but the one that worked for me is here:
https://developer.vuforia.com/forum/issues-and-bugs/camera-profile-folder-missing
Basically tells you that the 64bits version may cause this problem and you should use the 32 version.
If you need to download a previous version of Unity, this is the link where you can make it. https://unity3d.com/get-unity/download/archive.

Maybe we can combine the programs we have made on Unity to get a better solution that fits to the scope on our project.

Best Regards,
Francisco Andres Ortiz
